### PR TITLE
DROOLS-5419 : Scenario Coverage Report counts and shows previous names of renamed rules

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.redhat.com/browse/DROOLS-5419

### CODE REVIEW

**Comment by Rikkola** [comment]
@yesamer Hi, you might be the best to review this one.

File: kie-wb-common-services/kie-wb-common-services-backend/src/test/java/org/kie/workbench/common/services/backend/builder/core/BuilderRenameTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    @Test
+    public void noChanges() {
+
+        builder.applyBatchResourceChanges(new HashMap<>());
+
+        verify(kieFileSystem, never()).delete(any());
+    }
+
+    @Captor
+    private ArgumentCaptor<String> pathCaptor;
```
Comment (yesamer) [inline kie-wb-common-services/kie-wb-common-services-backend/src/test/java/org/kie/workbench/common/services/backend/builder/core/BuilderRenameTest.java:103]: @Rikkola Can you please move this field `pathCaptor` with other fields above?

File: kie-wb-common-services/kie-wb-common-services-backend/src/test/java/org/kie/workbench/common/services/backend/builder/core/BuilderRenameTest.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                              mock(LRUModuleDependenciesClassLoaderCache.class),
+                              mock(LRUPomModelCache.class),
+                              mock(PackageNameWhiteListService.class),
+                              mock(Predicate.class),
+                              mock(KieBuilder.class),
+                              kieFileSystem);
+    }
+
+    @After
+    public void tearDown() throws Exception {
```
Comment (yesamer) [inline kie-wb-common-services/kie-wb-common-services-backend/src/test/java/org/kie/workbench/common/services/backend/builder/core/BuilderRenameTest.java:90]: Are these `throws Exception` necessary?

**Comment by Rikkola** [comment]
@yesamer Thank you. I'll do the updates you requested.

**Comment by Rikkola** [comment]
kie-jpmml-integration errors, master likely out of synch. Jenkins please retest this.

**Comment by Rikkola** [comment]
jenkins please retest this. The cause of failure page gives 404.

**Comment by Rikkola** [comment]
Jenkins please retest this

**Comment by Rikkola** [comment]
Jenkins please retest this

**Comment by Rikkola** [comment]
@kiegroup/gatekeepers Can I get a merge please.

**Pull request merged**